test/parallel/test-cli-permission-deny-fs.js

```diff
@@ -47,6 +47,54 @@ const path = require('path');
   assert.strictEqual(status, 0);
 }
 
+{
+  const tmpPath = path.resolve('/tmp/');
+  const otherPath = path.resolve('/other-path/');
+  const { status, stdout } = spawnSync(
+    process.execPath,
+    [
+      '--experimental-permission',
+      '--allow-fs-write', `${tmpPath},${otherPath}`, '-e',
+      `console.log(process.permission.has("fs"));
+      console.log(process.permission.has("fs.read"));
+      console.log(process.permission.has("fs.write"));
+      console.log(process.permission.has("fs.write", "/tmp/"));
+      console.log(process.permission.has("fs.write", "/other-path/"));`,
+    ]
+  );
+  const [fs, fsIn, fsOut, fsOutAllowed1, fsOutAllowed2] = stdout.toString().split('\n');
+  assert.strictEqual(fs, 'false');
+  assert.strictEqual(fsIn, 'false');
+  assert.strictEqual(fsOut, 'false');
+  assert.strictEqual(fsOutAllowed1, 'true');
+  assert.strictEqual(fsOutAllowed2, 'true');
+  assert.strictEqual(status, 0);
+}
+
+{
```

Comment:
I'd also create: `test-permission-path-delimiter` that starts with:

```js
// Flags: --experimental-permission --allow-fs-read=* --allow-fs-write=/tmp/file,with,comma.txt --permission-fs-path-delimiter=;

...
```

Reply:
Do you mean a new test file or a new test case using those flags?

Reply:
A new test file

Reply:
I think I did what you meant, can you confirm?